.github/workflows/ci.yml

```
@@ -0,0 +1,33 @@
+name: Discussion Avatar CI - See Discussion Platform Actions
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Trigger workflow in discussion-platform
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.PAT_ACTIONS_DISCUSSION_PLATFORM }}
+          script: |
+            await github.rest.actions.createWorkflowDispatch({
+              owner: 'guardian',
+              repo: 'discussion-platform',
+              workflow_id: 'discussion-avatar-ci.yml',
+              ref: 'mm/avatar-api-gha', // TODO - change to main when discussion-platform PR is merged
```

```suggestion
              ref: 'main',
```

add this suggestion as a commit before we merge